### PR TITLE
Revise election app admin account topic, add images

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**

### CODE REVIEW

File: packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/03-admin-account.md [hunk 1]
```diff
@@ -1,151 +1,180 @@
...
-create and fund this account using the Kadena JavaScript client. You verified the creation
-of the account in `Accounts` view of Chainweaver and by calling the `details` function of the
-`coin` module on Devnet using the Kadena JavaScript client. The admin user will use its KDA
-to pay gas fees charged for deploying keysets, deploying smart contract modules and sending
-transactions to nominate candidates for the election. In the following chapter you will learn
-how to create a namespace in which you will later define an admin keyset and deploy your
-smart contracts.
+In this tutorial, you learned how to:
+
+- Generate a new public and secret key key for a Kadena account using Chainweaver.
```
Comment (jessevanmuijden) [inline packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/03-admin-account.md:175]: double 'key'

**Review by jessevanmuijden — APPROVED** [review]

**Review by sstraatemans — APPROVED** [review]

File: packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/03-admin-account.md [hunk 2]
```diff
@@ -1,151 +1,180 @@
...
+In fact, if you followed that tutorial, you already have a public and private key pair that you could use as the basis for your administrative account. 
+However, for demonstration purposes in this tutorial, let's create a new key in Chainweaver and use the new key as the basis for your Kadena account.
+
+To create a new key pair and account:
+
+1. Verify the development network is currently running on your local computer.
+
+2. Open Chainweaver.
+
+3. Select **devnet** from the network list.
```
Comment (EnoF) [inline packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/03-admin-account.md:53]: Just checking, but how to add a `devnet` to chainweaver is already described earlier? If so , please ignore this comment :)

File: packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/03-admin-account.md [hunk 3]
```diff
@@ -1,151 +1,180 @@
...
+
+4. Click **Keys** in the Chainweaver navigation panel.
+
+5. Click **Generate Key** to add a new public key to your list of public keys. 
+
+6. Click **Add k: Account**  for the new public key to add a new account to the list of accounts you are watching in Chainweaver.
```
Comment (EnoF) [inline packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/03-admin-account.md:59]: Not sure if we want to explain it here, or if it is already explained elsewhere, but users might be wondering how this `k:` is introduced.

File: packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/03-admin-account.md (hunk 2 above)
Comment (lsgunnlsgunn) [reply packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/03-admin-account.md:53]: this is covered in the previous tutorial, but good to point out the assumption I'm making..

File: packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/03-admin-account.md (hunk 3 above)
Comment (lsgunnlsgunn) [reply packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/03-admin-account.md:59]: it's a bit of a sticky wicket to explain here. I need to think more on it.

**Review by EnoF — APPROVED** [review]

**Review by Randynamic — APPROVED** [review]

**Comment by Randynamic** [comment]
@lsgunnlsgunn this can be merged

**Pull request merged**